File: SDKs/amazonaws.com/AWSMigrationHub/2017-05-31/go/pkg/models/shared/listcreatedartifactsrequest.go
package shared



type ListCreatedArtifactsRequest struct {
    MaxResults *int64 `json:"MaxResults,omitempty"`
    MigrationTaskName string `json:"MigrationTaskName"`
    NextToken *string `json:"NextToken,omitempty"`
    ProgressUpdateStream string `json:"ProgressUpdateStream"`
    
}

File: SDKs/amazonaws.com/AWSMigrationHub/2017-05-31/typescript/src/sdk/models/shared/listcreatedartifactsrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class ListCreatedArtifactsRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=MaxResults" })
  maxResults?: number;

  @SpeakeasyMetadata({ data: "json, name=MigrationTaskName" })
  migrationTaskName: string;

  @SpeakeasyMetadata({ data: "json, name=NextToken" })
  nextToken?: string;

  @SpeakeasyMetadata({ data: "json, name=ProgressUpdateStream" })
  progressUpdateStream: string;
}
